apk-generator/v2/app/utils/notification.py

```diff
@@ -21,7 +22,46 @@ def send(to, subject, message, file_attachment, via_api):
         :param via_api:
```

Comment:
This function is just for choosing which method to use. SMTP related config to be done in the smtp send method. Sendgrid config to be done sendgrid method.